docs/checks/internal/validate/main.go

```diff
@@ -70,6 +71,10 @@ func listCheckFiles() (map[string]string, error) {
 			continue
 		}
 
+		// WARNING: assume the filename is the same during migration
```

Comment:
Unrelated to this PR - can we do away with these validations where we string parse the checks and the API usage?

Reply:
Using a CFG is better but requires more work. If you have something specific in mind that's easy to implement, sure. The string based approach currently works, so I don't see this as a priority unless it keeps failing.
What do you have in mind?